docs/reference/migration/migrate_7_0/settings.asciidoc

```diff
@@ -36,6 +36,12 @@ of the node id. It can still be configured explicitly in `elasticsearch.yml`.
   available to keep the display output in APIs as `bulk` instead of `write`.
   These fallback settings and this system property have been removed.
 
+[float]
+==== Disabling memory-mapping
+
+* The deprecated `node.store.allow_mmapfs` setting has been removed in favour of
```

Comment:
I will create a separate PR on 6.x which introduces `node.store.allow_mmapfs` as a deprecated setting.

Reply:
I'm wondering that saying "renamed to" rather than "removed in favour of" might be clearer?

Reply:
Makes sense. I changed it now to

> The setting `node.store.allow_mmapfs` has been renamed to `node.store.allow_mmap`.